app/fastTrack/src/components/FastTrackModal.js

```diff
@@ -1,51 +1,79 @@
 // @flow strict
 
 import * as React from 'react';
-import { View } from 'react-native';
+import { View, WebView } from 'react-native';
```

Comment:
Please use `webview` from '@kiwicom/mobile-shared` instead. WebView from `react-native` is deprecated.